docs/api/client/hoodie.account.rst

```diff
@@ -20,11 +20,15 @@ Example
 
 .. code:: js
 
-    if (hoodie.account.isSignedIn()) {
-    renderWelcome(hoodie.account)
-    }
+    hoodie.account.get('session').then(function (sessionProperties) {
+      if (!sessionProperties) {
+        return redirectToHome();
+      }
+
+      renderWelcome(sessionProperties);
+    }).catch(redirectToHome);
 
-    hoodie.account.on('signout', redirectToHome)
+    hoodie.account.on('signout', redirectToHome);
```

Comment:
could you please remove the `;`s? We follow the [rules by Feross’ standard library](https://github.com/feross/standard#the-rules) throughout our codebase. I can also do it for you if you are busy, no problem :)